custom_components/opnsense/binary_sensor.py

```
@@ -24,131 +25,101 @@ async def async_setup_entry(
+            device_class=None,
+            # entity_category=entity_category,
+            entity_registry_enabled_default=False,
+        ),
+    )
+    entities.append(entity)
+
+    entity = OPNsensePendingNoticesPresentBinarySensor(
+        config_entry=config_entry,
+        coordinator=coordinator,
+        entity_description=BinarySensorEntityDescription(
+            key=f"notices.pending_notices_present",
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)